WordPress/gutenberg
# Tag Cloud: Add color block support

## What?

Add color block support to the `Tag Cloud` block.

Part of https://github.com/WordPress/gutenberg/issues/43245

## Why?
`Tag Cloud` block is missing color support.

## How?

Adds the color block support in block.json

## Testing Instructions
- Go to Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
- Make sure that `Tag Cloud` block's colors setting are configurable via Global Styles
- Verify that Global Styles are applied correctly in the editor and frontend
- Edit template/page, Add `Tag Cloud` block and Apply the border styles
- Verify that block styles take precedence over global styles
- Verify that block color styles display correctly in both the editor and frontend

## Screenshots or screencast 
In backend:
<img width="1214" alt="image" src="https://github.com/user-attachments/assets/149d9117-3f9d-4f6e-94bc-32bd9956588c">
 
In frontend:
<img width="780" alt="image" src="https://github.com/user-attachments/assets/82a49225-db84-48ce-b252-63688b965fc3">
